### PR TITLE
Document Instructions how to install kyma@v3 binary

### PR DESCRIPTION
**Pull request by kwiatekus**
**Description**

Changes proposed in this pull request:

- add a script that downloads latests kyma cli binary
- document installation steps

**Related issue(s)**
https://github.com/kyma-project/cli/issues/2142

### CODE REVIEW

File: hack/get-kyma-v3-alpha.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+mkdir tmp
+cd tmp
+
+curl -Lo kymav3.tar.gz https://github.com/kyma-project/cli/releases/download/v0.0.0-dev/kyma_Darwin_x86_64.tar.gz # kyma-linux, kyma-linux-arm, kyma.exe, or kyma-arm.exe
```
Comment (halamix2) [inline hack/get-kyma-v3-alpha.sh:5]: You can generalize this script for Mac/Linux:
```bash
OS="$(uname -s)" # e.g Darwin
ARCH="$(uname -m)" # e.g. arm64

curl -Lo kymav3.tar.gz https://github.com/kyma-project/cli/releases/download/v0.0.0-dev/kyma_${OS}_${ARCH}.tar.gz
```

File: hack/get-kyma-v3-alpha.sh
Comment (halamix2) [inline hack/get-kyma-v3-alpha.sh:1]: make sure you've ran `chmod +x hack/get-kyma-v3-alpha.sh` to skip one step on the user side

File: README.md [hunk 2]
```diff
@@ -9,6 +9,26 @@
...
+## Install 
+
+> [!WARNING]
+> `v3` is still at prototyping stage. All comands are still considered alpha - use it at you own risk.
+
+Download latest built from main branch from [v0.0.0-dev](https://github.com/kyma-project/cli/releases/tag/v0.0.0-dev) release assets.
+
+If you want to run kyma CLI on MacOS you can run the following script from the project's root folder:
+```sh
+ hack/get-kyma-v3-alpha.sh
```
Comment (halamix2) [inline README.md:21]:
```suggestion
 ./hack/get-kyma-v3-alpha.sh
```

File: hack/get-kyma-v3-alpha.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+
+tar zxvf kymav3.tar.gz 
+cp kyma ../bin/kyma@v3
+cp kyma /usr/local/bin/kyma@v3
```
Comment (halamix2) [inline hack/get-kyma-v3-alpha.sh:10]: copying to /usr might require root permissions

File: hack/get-kyma-v3-alpha.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+cd tmp
+
+OS="$(uname -s)" # e.g Darwin
+ARCH="$(uname -m)" # e.g. arm64
+
+curl -Lo kymav3.tar.gz https://github.com/kyma-project/cli/releases/download/v0.0.0-dev/kyma_${OS}_${ARCH}.tar.gz
+
+
+tar zxvf kymav3.tar.gz 
+cp kyma ../bin/kyma@v3
```
Comment (halamix2) [inline hack/get-kyma-v3-alpha.sh:12]:
```suggestion
mkdir -p ../bin
cp kyma ../bin/kyma@v3
```

File: README.md [hunk 5]
```diff
@@ -9,6 +9,27 @@
...
+```sh
+ ./hack/get-kyma-v3-alpha.sh
+```
+
+The script above will download a Linux/MacOS variant of kyma CLI v3 binary into `bin/kyma@v3`.
+
+Inspect new available alpha commands by calling `--help` option: 
+
+```sh
+ bin/kyma@v3 alpha  --help
```
Comment (halamix2) [inline README.md:29]:
```suggestion
 ./bin/kyma@v3 alpha  --help
```

File: README.md [hunk 6]
```diff
@@ -19,7 +40,7 @@ Kyma CLI is a command line tool which supports [Kyma](https://github.com/kyma-pr
 ```
 docker pull kennethreitz/httpbin
 
-go run main.go image-import kennethreitz/httpbin:latest
+kyma@v3 alpha image-import kennethreitz/httpbin:latest
```
Comment (halamix2) [inline README.md:43]:
```suggestion
./bin/kyma@v3 alpha image-import kennethreitz/httpbin:latest
```

File: README.md [hunk 7]
```diff
@@ -9,6 +9,27 @@
 
 Kyma CLI is a command line tool which supports [Kyma](https://github.com/kyma-project/kyma) users.
 
+## Install 
+
+> [!WARNING]
+> `v3` is still at prototyping stage. All comands are still considered alpha - use it at you own risk.
```
Comment (grego952) [inline README.md:15]:
```suggestion
> `v3` is still in the prototyping stage. All commands are still considered alpha - use it at your own risk.
```

File: README.md [hunk 8]
```diff
@@ -9,6 +9,27 @@
...
+
+Download latest built from main branch from [v0.0.0-dev](https://github.com/kyma-project/cli/releases/tag/v0.0.0-dev) release assets.
```
Comment (grego952) [inline README.md:17]:
```suggestion
Download the latest build from the main branch from [v0.0.0-dev](https://github.com/kyma-project/cli/releases/tag/v0.0.0-dev) release assets.
```

File: README.md [hunk 9]
```diff
@@ -9,6 +9,27 @@
...
+
+If you want to run kyma CLI on MacOS you can run the following script from the project's root folder:
```
Comment (grego952) [inline README.md:19]:
```suggestion
To run Kyma CLI on MacOS, run the following script from the project's root folder:
```

File: README.md [hunk 10]
```diff
@@ -9,6 +9,27 @@
...
+```sh
+ ./hack/get-kyma-v3-alpha.sh
+```
+
+The script above will download a Linux/MacOS variant of kyma CLI v3 binary into `bin/kyma@v3`.
```
Comment (grego952) [inline README.md:24]:
```suggestion
This script downloads a Linux/MacOS variant of Kyma CLI v3 binary into `bin/kyma@v3`.
```

File: README.md [hunk 11]
```diff
@@ -9,6 +9,27 @@
...
+
+Inspect new available alpha commands by calling `--help` option:
```
Comment (grego952) [inline README.md:26]:
```suggestion
Inspect the new available alpha commands by calling the `--help` option: 
```

**Review by halamix2 — APPROVED** [review]

**Review by grego952 — APPROVED** [review]

**Pull request merged**